tests/test_slash_list.py

```diff
@@ -180,5 +180,42 @@ def test_slash_list_with_filtering(tmpdir):
     assert listed_tests == ['{}:{}'.format(fp.name, test_name) for test_name in ('test_a', 'test_c')]
 
 
+def test_slash_list_with_parameterization_filtering(tmpdir):
+    from textwrap import dedent
+    with tmpdir.join('test_file.py').open('w') as fp:
+        _print = functools.partial(print, file=fp)
+        _print(dedent("""
+            import slash
+            @slash.parametrize('x', [
```

Comment:
We have a better facility for generating and then asserting what runs -- take a look at https://github.com/getslash/slash/blob/a699ea65ea14d98d6cb5646bd29a0e6423cce2df/tests/test_parametrization_labels.py#L34 for an example on how it can be used

Reply:
I saw it, but wasn't sure how to specify `-k` filters to `suite_builder.build().run()`

Reply:
It's ok to extend `run` to accept additional args. It doesn't accept them at the moment simply because it wasn't needed thus far.